juniper_official/routing/collect-sr-color-lsp-stats.rule

```diff
@@ -63,7 +89,37 @@ healthbot {
                     }
                 }
                 type string;
-                description "LSP name";
+                description "LSP computed name";
+            }
+            field lsp-cfg-name {
+                sensor jnpr-sr-color-lsp {
+                    path juniper_interface_name_dummy_path;
+                    data-if-missing {
+                        value "jnpr_sr_lsp";
+                    }
+                }
+                sensor cisco-oc-sr-color-lsp {
+                    where "/interfaces/interface/@name =~ /{{input-cisco-sr-color-lsp-name}}/";
+                    path "/interfaces/interface/@name";
+                }                
+                type string;
+                description "Interface name";
+            }
+            field vendor {
```

Comment:
Please use the vendor from the inventory and not overload the telemetry processing